redwood-widget-composeui/src/commonMain/kotlin/app/cash/redwood/widget/composeui/insets.kt

```diff
@@ -13,11 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package app.cash.redwood.treehouse.composeui
+package app.cash.redwood.widget.composeui
 
 import androidx.compose.runtime.Composable
 import app.cash.redwood.ui.Margin
 
 /** Return the device's insets in screen density-independent pixels. */
 @Composable
-internal expect fun safeAreaInsets(): Margin
+public expect fun safeAreaInsets(): Margin
```

Comment:
Does this revert once `RedwoodContent` exists in this module?

Reply:
After looking at the other PR, I think this should move into `redwood-composeui` and not into a new module.

Reply:
> Does this revert once RedwoodContent exists in this module?

Nope, cause we need it for `TreehouseContent`. Though if we make `TreehouseContent` a wrapper around `RedwoodContent`, we could revert it.

Reply:
I think long term that should be the goal, but it's hard to reason about it right now. We can leave it as-is while we figure it out.